packages/maskbook/src/plugins/ITO/SNSAdaptor/PostInspector.tsx

```diff
@@ -51,7 +51,7 @@ export function PostInspector(props: PostInspectorProps) {
                 (t) =>
                     ({
                         address: t.address,
-                        type: isSameAddress(t.address, NATIVE_TOKEN_ADDRESS)
+                        type: isSameAddress(t.address, NATIVE_TOKEN_ADDRESS ?? '')
```

Comment:
Why will `NATIVE_TOKEN_ADDRESS` be yield to `undefined`?

Reply:
why not use `isNative`

https://github.com/DimensionDev/Maskbook/blob/41767116e15729dd826d042c6afe4805618e58fc/packages/web3-shared/src/utils/address.ts#L22